### DISCUSSION TITLE
Flutter Auth Error - Quickstart Guide

### DISCUSSION THREAD

**Discussion by angus-clark** [supabase/supabase#4953] [Questions] [answered]
I followed the quick start guide entirely, however, when attempting to login the following error occurs. Supabase successfully logs that a login event occurred, but the screen does not update to the profile page.

Is there something I am missing or is there an issue with the package?

`E/flutter ( 6946): [ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: LateInitializationError: Field 'changedAccessToken' has not been initialized.
E/flutter ( 6946): #0      SupabaseClient.changedAccessToken (package:supabase/src/supabase.dart)
E/flutter ( 6946): #1      SupabaseClient._handleTokenChanged (package:supabase/src/supabase.dart:171:46)
E/flutter ( 6946): #2      SupabaseClient._listenForAuthEvents.<anonymous closure> (package:supabase/src/supabase.dart:165:7)
E/flutter ( 6946): #3      GoTrueClient._notifyAllSubscribers.<anonymous closure> (package:gotrue/src/gotrue_client.dart:442:53)
E/flutter ( 6946): #4      _LinkedHashMapMixin.forEach (dart:collection-patch/compact_hash.dart:539:8)
E/flutter ( 6946): #5      GoTrueClient._notifyAllSubscribers (package:gotrue/src/gotrue_client.dart:442:25)
E/flutter ( 6946): #6      GoTrueClient.getSessionFromUrl (package:gotrue/src/gotrue_client.dart:241:7)
E/flutter ( 6946): <asynchronous suspension>
E/flutter ( 6946): #7      SupabaseAuthState.recoverSessionFromUrl (package:supabase_flutter/src/supabase_auth_state.dart:67:22)
E/flutter ( 6946): <asynchronous suspension>
E/flutter ( 6946): `

**Comment by dshukertjr** [comment, marked as answer]
Hi @clarky2233 !

I think this issue has been fixed with the newest version of `supabase_flutter` with [this PR](https://github.com/supabase-community/supabase-dart/pull/84/files). Could you upgrade your `supabase_flutter` version to the newest one and give it another try?

**Reply by angus-clark** [reply]
Thank you I'll check it out. Cheers for the awesome work!
